test/PoolManager.t.sol

```diff
@@ -741,13 +745,13 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         snapEnd();
     }
 
-    function test_swap_accruesProtocolFees(uint8 protocolFee1, uint8 protocolFee0) public {
-        protocolFee0 = uint8(bound(protocolFee0, 4, type(uint8).max));
-        protocolFee1 = uint8(bound(protocolFee1, 4, type(uint8).max));
+    function test_swap_accruesProtocolFees(uint16 protocolFee0, uint16 protocolFee1) public {
+        protocolFee0 = uint16(bound(protocolFee0, 1, 2500));
+        protocolFee1 = uint16(bound(protocolFee1, 1, 2500));
 
-        uint16 protocolFee = (uint16(protocolFee1) << 8) | (uint16(protocolFee0) & uint16(0xFF));
+        uint24 protocolFee = (uint24(protocolFee1) << 12) | (uint24(protocolFee0) & uint24(0x1000));
```

Comment:
`uint24(protocolFee0) & uint24(0x1000)`

that feels wrong to me - isnt that canceling out the protocol fee? like if protocolFee0 is `1` then `uint24(protocolFee0) & uint24(0x1000)` is going to be 0?

Reply:
i think you can just remove the &

Reply:
Isn't this just clearing the extra bits? we're fuzzing with uint16s but they each should be 12 bits so they pack to a uint24

Reply:
oh hm so it should be `& 0xFFF` no?

Reply:
but it already does a `bound` above so i dont think any masking is needed

Reply:
oh yeah i think i just need to remove the &